src/dvc_render/vega_templates.py

```diff
@@ -610,15 +492,53 @@ class LinearTemplate(Template):
                 "title": Template.anchor("y_label"),
                 "scale": {"zero": False},
             },
-            "color": {"field": "rev", "type": "nominal"},
+            "color": {
+                "field": "rev",
+                "type": "nominal",
+                "legend": {"symbolOpacity": 1},
+            },
         },
+        "params": [
+            {
+                "name": "opacity",
+                "value": 0.1,
+                "bind": {
+                    "input": "range",
+                    "min": 0,
+                    "max": 1,
+                    "step": 0.1,
+                },
+            },
+            {
+                "name": "smooth",
+                "value": 0.001,
+                "bind": {
+                    "input": "range",
+                    "min": 0.001,
+                    "max": 1,
+                    "step": 0.001,
+                },
+            },
+        ],
+        "transform": [
```

Comment:
one downside of complicating it this way that other values (like `threshold` values won't go through the transform) and won't be part of the tooltips. 

Also, we should be aware that transform complicates it a bit, makes it harder to take this as a baseline and change to your needs, etc, etc.

Reply:
> Also, we should be aware that transform complicates it a bit, makes it harder to take this as a baseline and change to your needs, etc, etc.

I'm not sure if that many people to go that path though. So probably not worth over-optimizing it.

Reply:
> one downside of complicating it this way that other values (like `threshold` values won't go through the transform) and won't be part of the tooltips.

That's a good point since we just added that per a feature request. Let's think if there's any way to preserve it or any better ideas for the template.

> I'm not sure if that many people to go that path though. So probably not worth over-optimizing it.

Agreed. My thinking here was that we should make the default template as good and useful as possible without worrying much about how complex it is. Seems like better UX/less friction to have these sliders than to ask users to find the json template, edit it correctly, and then update dvc.yaml to use it.

We recently dropped the simple template because it was redundant, but I don't mind adding it back here if we are worried about giving power users a good starting point for custom templates.

The other drawback I see to this added complexity is that the parameters don't get codified anywhere. However, it doesn't worry me too much because:
1. Exporting to png at least captures the current state of the sliders, so at least it's possible to share the exact state of the plots.
2. It should be possible to save the current state of the sliders back into a custom template, so we could have an option to persist whatever changes are made on the frontend.